packages/cubejs-schema-compiler/src/compiler/CubeValidator.js

```diff
@@ -5,7 +5,7 @@ const identifierRegex = /^[_a-zA-Z][_a-zA-Z0-9]*$/;
 const identifier = Joi.string().regex(identifierRegex, 'identifier');
 const timeInterval =
```

Comment:
I am not 100% sure, but I think timeInterval & everyInterval will not work because it's needed to change `BaseQuery.parseSecondDuration` to support it.

Reply:
I'll have a look at this and see what might need to be added.

Reply:
This seems to be only for refresh key

I'm not trying to add quarter as a refresh key.

That's a separate feature/fix, which has the simple workaround to just use 90 days for your refresh key.

Reply:
@ovr